src/sagas/user.js

```diff
@@ -0,0 +1,32 @@
+import { take, select, all, put } from 'redux-saga/effects';
+import { selectCurrentAccount, changePassword } from '../reducers/Account';
+import { FORM_SUBMIT_START, selectForm, resetForm, formSubmitFail, formSubmitSuccess } from '../reducers/form';
+import { FORM_NAME } from '../containers/PasswordChangeModal/PasswordChangeModal';
+
+export function* passwordChangeSaga() {
+  while (1) {
+    const action = yield take(FORM_SUBMIT_START);
+    const formName = action.payload;
+    if (formName !== FORM_NAME) {
+      return;
+    }
+    const form = yield select(selectForm, formName);
+    const { oldPassword, newPassword, confirmPassword } = form.data;
+    const account = yield select(selectCurrentAccount);
+    if (newPassword !== confirmPassword) {
+      yield put(formSubmitFail(formName, 'Confirm password not match'));
```

Comment:
Dùng continue để đở if else lồng nhau

```suggestion
yield put(formSubmitFail(formName, 'Confirm password not match'));
continue;
```